### PR TITLE
l10n : First test 

### PR DESCRIPTION
**Pull request by hernic**
Does this deserve a new branch for a first review ?

### CODE REVIEW

**Comment by hernic** [comment]
sorry for the inappropriate title, it is due to a handling error

File: build.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 			optimize="true"
 			debug="true"
 			source="1.6" 
-			target="1.6">
+			target="1.7">
```
Comment (Thynix) [inline build.xml]: 1468 will still be on Java 1.6, so this shouldn't be updated quite yet. I can cherry-pick and avoid this one, so no need to do anything.

File: src/plugins/Freereader/Freereader.java [hunk 2]
```diff
@@ -75,12 +81,71 @@ public void terminate()
 		Logger.normal(this, pluginName + " terminated.");
 	}
 	
-	public String getString(String key) {
-		// TODO return a translated string
-		return key;
+	/**
+	 * This code is only used by FredPluginL10n...
```
Comment (Thynix) [inline src/plugins/Freereader/Freereader.java]: What is this comment for?

Comment (hernic) [inline src/plugins/Freereader/Freereader.java]: Simply specify than this call should not be used outside of l10n logic.
- WoT : no comment - https://github.com/freenet/plugin-WebOfTrust/blob/master/src/plugins/WebOfTrust/WebOfTrust.java#L3832
- FlogHelper : same comment - https://github.com/freenet/plugin-FlogHelper/blob/master/src/plugins/floghelper/FlogHelper.java#L435
- Freetalk : same comment - https://github.com/freenet/plugin-Freetalk/blob/master/src/plugins/Freetalk/Freetalk.java#L783

... remove or not remove, change comment or leaving like this ?

Comment (Thynix) [inline src/plugins/Freereader/Freereader.java]: Oh, if there's that much president might as well remain consistent for now
then. It seems fine to leave it like that. I will test/merge this and add
it to Transifex when I next get a block of time for Freenet. Thanks!

**Comment by Thynix** [comment]
Thanks!

**Comment by Thynix** [comment]
I've added this to Transifex.

**Comment by ArneBab** [comment]
I compiled the current master, but it only shows the L10n strings, not the actual translations. I see “Common.AddRemoveFeeds” as title instead of “Add and Remove Feeds”.

**Comment by Thynix** [comment]
Is it lacking a fallback language - does it work when set to English or
French - or is this more fundamental? I did not test this because it looked
fine in review and I assumed Nicolas tested it; maybe I should have tested
it anyway.

**Comment by hernic** [comment]
It works fine on my windows box.

i check on a unbuntu one.

- Nicolas Hernandez
a-n - aleph-networks
*CEO*
http://www.aleph-networks.com

**Comment by ArneBab** [comment]
I’m using an english locale for fred, but my fred has a few local modifications (purge-db4o plus the search box removal, I think).

**Comment by xor-freenet** [comment]
I suggest reopening, this is bugged:

[12:40] <xor-freenet> salvatoreG: the problem likely is that this doesn't include the l10n files: https://github.com/freenet/plugin-Freereader/blob/348e094ee4049fbaffd6bf2f4bf8738616336e9c/build.xml#L46
[12:42] <xor-freenet> salvatoreG: WoT does it like that: https://github.com/freenet/plugin-WebOfTrust/blob/a75200950118b53a0d276ca02ba274645ea42804/build.xml#L107
[12:46] <xor-freenet> salvatoreG: so in the case of freereader, you'd make build.xml copy the l10n files to ${build.classes}
[12:46] <xor-freenet> salvatoreG: where WoT copies to <copy todir="${build}">

**Comment by xor-freenet** [comment]
I suggest reopening, this is bugged:

[12:40] <xor-freenet> salvatoreG: the problem likely is that this doesn't include the l10n files: https://github.com/freenet/plugin-Freereader/blob/348e094ee4049fbaffd6bf2f4bf8738616336e9c/build.xml#L46
[12:42] <xor-freenet> salvatoreG: WoT does it like that: https://github.com/freenet/plugin-WebOfTrust/blob/a75200950118b53a0d276ca02ba274645ea42804/build.xml#L107
[12:46] <xor-freenet> salvatoreG: so in the case of freereader, you'd make build.xml copy the l10n files to ${build.classes}
[12:46] <xor-freenet> salvatoreG: where WoT copies to ```<copy todir="${build}">```

**Comment by xor-freenet** [comment]
FYI for the original author of the pull request: salvatoreG maybe will fix this. So maybe talk to him on IRC before dealing with it yourself.

**Pull request merged**